examples/text_labels_and_annotations/demo_text_rotation_mode.py

```diff
@@ -3,7 +3,14 @@
 Demo Text Rotation Mode
 =======================
 
+The axes method text takes an argument rotation_mode that controls the
```

Comment:
You format the name of methods to directly link to the respective method, and format other code in monospace font, e.g.

```suggestion
The axes' method `.axes.Axes.text` takes an argument ``rotation_mode`` that controls the
```

Reply:
Thanks! I'll make that change everywhere in the docstring.